test/io/ferguson/hash_group.good

```diff
@@ -0,0 +1,10 @@
+  1
+  1.
+  1..
+  1.10
+  1. 2
+  1. 2
+  1  2
```

Comment:
Yep. The format is ### - (eg for the first one) meaning it should print out 3 columns at least...